Fix broken autoreconf

Upstream: https://git.xiph.org/?p=theora.git;a=commitdiff;h=28cc6dbd9b2a141df94f60993256a5fca368fa54

Signed-off-by: Bernd Kuhls <bernd.kuhls@t-online.de>
[Dario: make the patch to be applied with fuzz factor 0]
Signed-off-by: Dario Binacchi <dario.binacchi@amarulasolutions.com>


From: Tim Terriberry <tterribe@xiph.org>
Date: Fri, 20 May 2011 20:41:50 +0000 (+0000)
Subject: Make autoreconf -i -f work.
X-Git-Url: https://git.xiph.org/?p=theora.git;a=commitdiff_plain;h=28cc6dbd9b2a141df94f60993256a5fca368fa54

Make autoreconf -i -f work.

Patch from David Schleef.

svn path=/trunk/theora/; revision=17990
---

diff --git a/Makefile.am b/Makefile.am
index 89fd753..1783857 100644
--- a/Makefile.am
+++ b/Makefile.am
@@ -2,5 +2,7 @@
 AUTOMAKE_OPTIONS = foreign 1.6 dist-zip dist-bzip2
 
+ACLOCAL_AMFLAGS=-I m4
+
 if THEORA_ENABLE_EXAMPLES
 EXAMPLES_DIR = examples
 else
diff --git a/configure.ac b/configure.ac
index 1cbec1a..456b603 100644
--- a/configure.ac
+++ b/configure.ac
@@ -61,7 +61,7 @@ AC_LIBTOOL_WIN32_DLL
 AM_PROG_LIBTOOL
 
 dnl Add parameters for aclocal
-AC_SUBST(ACLOCAL_AMFLAGS, "-I m4")
+AC_CONFIG_MACRO_DIR([m4])
 
 dnl Check for doxygen
 AC_CHECK_PROG(HAVE_DOXYGEN, doxygen, true, false)
